lib/Mixpanel.php

```diff
@@ -149,10 +149,10 @@ public function __construct($token, $options = array()) {
      * @return Mixpanel
      */
     public static function getInstance($token, $options = array()) {
-        if(!isset(self::$_instance)) {
-            self::$_instance = new Mixpanel($token, $options);
+        if(!isset(self::$_instances[$token])) {
```

Comment:
Let's use array_key_exists to check. IIRC, this will throw a notice if the key isn't in there.